CoreFoundation/PlugIn.subproj/CFBundle_InfoPlist.c

```diff
@@ -17,7 +17,7 @@
 
 #if (TARGET_OS_MAC || TARGET_OS_LINUX || TARGET_OS_BSD) && !TARGET_OS_CYGWIN
 #include <dirent.h>
-#if !TARGET_OS_ANDROID
+#if TARGET_OS_MAC || TARGET_OS_BSD
```

Comment:
The call site for `sysctlbyname` is guarded by `TARGET_OS_IPHONE`, so perhaps use that here, too?

Reply:
Well, `TARGET_OS_IPHONE` wasn't included in the original code, I'm leery of adding anything that could cause unintended side effects.